### PR TITLE
Clims 45 implement task list

### PR DESCRIPTION
**Pull request by johandahlberg**
Implement the basic stuff to get the task list API going as expected. This included and endpoint, and a serializer for the model. After me and @withrocks discussed this we decided that we would go with the default Django serializers rather than using the Sentry "register magic". Going of the Zen "explicit it better that explicit".

I've also started moving some (though not all) of the relevant code into it's own clims namespace.

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    // for the documentation about the extensions.json format
-    "recommendations": [
-        "dzannotti.vscode-babel-coloring",
-        "esbenp.prettier-vscode",
-        "ms-python.python",
-        "dbaeumer.vscode-eslint",
-        "lextudio.restructuredtext",
-        "ziyasal.vscode-open-in-github"
-    ]
+  // See https://go.microsoft.com/fwlink/?LinkId=827846
```
Comment (johandahlberg) [inline .vscode/extensions.json:2]: We need to figure out how to not get this to happen (probably by placing auto-formatting settings in the repo).

File: .vscode/settings.json [hunk 2]
```diff
@@ -46,17 +46,15 @@
...
   "python.formatting.autopep8Args": ["--global-config", "${workspaceRoot}/setup.cfg"],
   // https://github.com/DonJayamanne/pythonVSCode/issues/992
-  "python.pythonPath": "/home/stest683/.virtualenvs/clims2/bin/python",
+  "python.pythonPath": "/home/johda411-local/anaconda3/envs/commonlims2/bin/python",
   // test discovery is sluggish and the UI around running
   // tests is often in your way and misclicked
-  "python.unitTest.pyTestEnabled": true,
-  "python.unitTest.unittestEnabled": false,
-  "python.unitTest.nosetestsEnabled": false,
+  "python.testing.pyTestEnabled": true,
```
Comment (johandahlberg) [inline .vscode/settings.json:52]: This is apparently the new way that this should be setup.

File: src/sentry/api/endpoints/user_task.py [hunk 3]
```diff
@@ -4,561 +4,11 @@
 from sentry.api.bases.user_task import UserTaskBaseEndpoint
 from sentry.api.bases.organization import OrganizationEndpoint
 from clims.models import UserTask
+from clims.api.serializers.models.usertask import UserTaskSerializer
 from sentry.models.activity import Activity
-from sentry.api.serializers import serialize
 from sentry.api.paginator import OffsetPaginator
 
 
-def update_user_tasks(request, organization_id, search_fn):
```
Comment (johandahlberg) [inline src/sentry/api/endpoints/user_task.py:12]: This is just removing old copy-paste-code from the PoC.

File: .vscode/launch.json [hunk 4]
```diff
@@ -1,36 +1,34 @@
...
+      "url": "http://localhost:8000",
+      "webRoot": "${workspaceRoot}/src/sentry/static/sentry/app"
+    },
+    {
+      "name": "sentry backend",
+      "type": "python",
+      "request": "launch",
+      "stopOnEntry": true,
+      // Replace with the full path to your lims (TODO: There must a variable we can use for this)
+      "program": "/home/johda411-local/anaconda3/envs/commonlims2/bin/lims",
```
Comment (withrocks) [inline .vscode/launch.json]: I wonder if we should just keep this `lims` or some other placeholder?

**Comment by withrocks** [comment]
Looks good!

**Pull request merged**